gslib/commands/du.py

```diff
@@ -48,68 +48,66 @@
+  For example, the following command reports the total space used by all objects and
+  subdirectories under gs://your-bucket/dir:
 
     gsutil du -s gs://your-bucket/dir
 
-  reports the total space used by all objects under gs://your-bucket/dir and
-  any sub-directories.
-
 
 <B>OPTIONS</B>
-  -0          Ends each output line with a 0 byte rather than a newline. This
-              can be useful to make the output more easily machine-readable.
+  -0          Ends each output line with a 0 byte rather than a newline. You
+              can use this to make the output machine-readable.
 
-  -a          Includes non-current object versions / generations in the listing
-              (only useful with a versioning-enabled bucket). Also prints
-              generation and metageneration for each listed object.
+  -a          Includes noncurrent object versions for a bucket with Object
+              Versioning enabled. Also prints the
+              generation and metageneration number for each listed object.
 
-  -c          Includes a grand total at the end of the output.
+  -c          Includes a total size at the end of the output.
 
-  -e          A pattern to exclude from reporting. Example: -e "*.o" would
-              exclude any object that ends in ".o". Can be specified multiple
+  -e          Exclude a pattern from the report. Example: -e "*.o"
+              excludes any object that ends in ".o". Can be specified multiple
               times.
 
-  -h          Prints object sizes in human-readable format (e.g., 1 KiB,
-              234 MiB, 2GiB, etc.)
+  -h          Prints object sizes in human-readable format. For example, ``1 KiB``,
+              ``234 MiB``, or ``2GiB``.
 
-  -s          Displays only the grand total for each argument.
+  -s          Displays only the total size for each argument.
 
-  -X          Similar to -e, but excludes patterns from the given file. The
-              patterns to exclude should be one per line.
+  -X          Similar to ``-e``, but excludes patterns from the given file. The
+              patterns to exclude should be listed one per line.
 
 
 <B>EXAMPLES</B>
-  To list the size of all objects in a bucket:
+  To list the size of each object in a bucket:
 
     gsutil du gs://bucketname
 
-  To list the size of all objects underneath a prefix:
+  To list the size of each object in the ``prefix`` subdirectory:
 
     gsutil du gs://bucketname/prefix/*
 
-  To print the total number of bytes in a bucket, in human-readable form:
+  To print the total number of bytes in a bucket in human-readable form:
 
     gsutil du -ch gs://bucketname
 
-  To see a summary of the total bytes in the two given buckets:
+  To see a summary of the total number of bytes in two given buckets:
 
     gsutil du -s gs://bucket1 gs://bucket2
 
-  To list the size of all objects in a versioned bucket, including objects that
-  are not the latest:
+  To list the size of each object in a bucket with Object Versioning
```

Comment:
The existing wording is not great, but it may be deliberate: this flag not only applies to buckets that have Object Versioning *currently* enabled, but I believe it also applies to buckets that had Object Versioning enabled *at some point in the past*.